script/Arcueid/Day 10/10_00_ARC10_5C.txt

```diff
@@ -574,8 +618,9 @@ As a normal person, I would really prefer not to have any more of these abnormal
 [sha:d8b769c67665884bf9391fd89f1153802ef04c8f]{
 -- Page 547, Offset 25679.
 -- 吸血鬼……死徒の天敵というのは、あなたたち人間そのものなんだから」
-// 吸血鬼 dropped here because its unnecessary
-The natural enemy of Dead Apostles... is a human, like yourself."
+// Not sure if this is supposed to be a subtle correction (not all vampires are dead apostles)
```

Comment:
I don't think so. this might be worth a lore understander's eyes but I thought it was just ugly repetition.

Reply:
I think I lean in favour of keeping both, because it doesn't really slow the TL down too much, and punts the job of interpretation to the reader.